packages/react/src/PageHeader/PageHeader.figma.tsx

```diff
@@ -2,15 +2,177 @@ import React from 'react'
 import {PageHeader} from '../'
 import figma from '@figma/code-connect'
 
-figma.connect(PageHeader, 'https://www.figma.com/design/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=38380%3A2926', {
-  props: {
-    // No matching props could be found for these Figma properties:
-    description: figma.boolean('hasDescription', {
-      false: undefined,
-      true: figma.children('_PageHeader.Description'),
-    }),
-    // "hasNavigation": figma.boolean('hasNavigation'),
-    // "hasContextArea": figma.boolean('hasContextArea')
-  },
-  example: ({description}) => <PageHeader>{description}</PageHeader>,
-})
+figma.connect(
+  PageHeader,
+  'https://www.figma.com/design/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=40002-2161&t=IXcS4U90x9HnYx6s-4',
+  {
+    props: {
+      // No matching props could be found for these Figma properties:
+      description: figma.children('_PageHeader.Description'),
+      titleArea: figma.children('_PageHeader.TitleArea'),
+      hasBorder: figma.boolean('hasBorder?'),
+      navigation: figma.children('_PageHeader.Navigation'),
+      contextArea: figma.children('_PageHeader.ContextArea'),
+    },
+    example: ({description, hasBorder, titleArea, navigation, contextArea}) => (
+      <PageHeader hasBorder={hasBorder}>
```

Comment:
It looks like TypeScript is complaining about `hasBorder` here, what is the corresponding prop for this in React? I couldn't seem to find a reference for borders in PageHeader 🤔